src/mode/remapper.ts

```diff
@@ -120,8 +125,17 @@ class Remapper {
       }
 
       vimState.isCurrentlyPerformingRemapping = false;
-
       return true;
+    } else {
+      // Check to see if a remapping could potentially be applied when more keys are received
+      for (let remap of this._remappings) {
```

Comment:
I see what you mean, it would be annoying to thread through. OK, a getter is fine.